api/v1alpha1/envoyproxy_types.go

```diff
@@ -86,8 +86,46 @@ type EnvoyProxySpec struct {
 	//
 	// +optional
 	Shutdown *ShutdownConfig `json:"shutdown,omitempty"`
+
+	// FilterOrder defines the order of filters in the Envoy proxy's HTTP filter chain.
+	//
+	// +optional
+	FilterOrder []FilterOrder `json:"filterOrder,omitempty"`
+}
+
+// FilterOrder defines the order of filters in the Envoy proxy's HTTP filter chain.
+// The order of filters is determined by the order field, where the filter with
+// the lowest order value is applied first.
+// If unspecified, the default order of filters is applied.
+// Default order of filters:
```

Comment:
did you pick this up from https://github.com/envoyproxy/go-control-plane/blob/main/pkg/wellknown/wellknown.go ?

Reply:
Yes. :-)